src/scripts/eos-analysis

```diff
@@ -219,6 +219,39 @@ The output file will be stored in EOS_BASE_DIRECTORY/POSTERIOR/pmc.
     parser_sample_pmc.set_defaults(cmd = cmd_sample_pmc)
 
 
+    # sample-nested
+    parser_sample_nested = subparsers.add_parser('sample-nested',
+        parents = [common_subparser],
+        description =
+'''
+Samples from a likelihood associated with a named posterior using dynamic nested sampling.
+
+The output file will be stored in EOS_BASE_DIRECTORY/POSTERIOR/nested.
+''',
+        help = 'Samples from a likelihood (associated with a posterior) using dynamic nested sampling.'
+    )
+    parser_sample_pmc.add_argument('posterior', metavar = 'POSTERIOR',
```

Comment:
You forgot to adjust the name of the object! All ``parser_sample_pmc`` should be ``parser_sample_nested`` here.